api/v1alpha1/batchrelease_plan_types.go

```diff
@@ -44,6 +44,13 @@ type ReleasePlan struct {
 	BatchPartition *int32 `json:"batchPartition,omitempty"`
 	// RolloutID indicates an id for each rollout progress
 	RolloutID string `json:"rolloutID,omitempty"`
+	// ToleratedFailedReplicas allow users ignore some pods that are not available due to some reasons,
+	// such as insufficient resources. For example, there are 10 pods were updated, and the
+	// ToleratedFailedReplicas=2, as long as 8 or more pods are available, we think that the
+	// rollout progression can be continued.
+	// If ToleratedFailedReplicas is not set (be nil), we will reuse `MaxUnavailable` field in workload
+	// to tolerate failed replicas by default.
+	ToleratedFailedReplicas *intstr.IntOrString `json:"toleratedFailedReplicas,omitempty"`
```

Comment:
Is it to define a unified structure, the same field does not need to be defined twice